### PR TITLE
[unifi-poller] convert to common library

### PR DESCRIPTION
**Pull request by onedr0p**
#### Special notes for your reviewer:

@bjw-s might need you assistant here on the servicemonitor portion.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

**Review by bjw-s — COMMENTED** [review]
As for the serviceMonitor, I believe this should be it:

```yaml
{{- if .Values.prometheus.serviceMonitor.enabled }}
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: {{ template "common.names.fullname" . }}
  labels:
    {{- include "common.labels" . | nindent 4 }}
    {{- with .Values.prometheus.serviceMonitor.additionalLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
spec:
  selector:
    matchLabels:
      {{- include "common.labels.selectorLabels" . | nindent 6 }}
  endpoints:
  - port: http
    {{- with .Values.prometheus.serviceMonitor.interval }}
    interval: {{ . }}
    {{- end }}
    path: /metrics
{{- end }} 
 ```

**Comment by onedr0p** [comment]
Looks good to me, this really helps me understand more :)

File: charts/unifi-poller/templates/configmap.yaml [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
   labels:
-{{ include "unifi-poller.labels" . | indent 4 }}
+    {{- include "common.labels" . | nindent 4 }}
 data:
-  up.yaml: {{ .Values.config | toYaml | quote }}
-
+  unifi-poller.yaml: |
+{{- with .Values.config }}
+    {{- toYaml . | nindent 4 }}
+{{- end }}
```
Comment (bjw-s) [inline charts/unifi-poller/templates/configmap.yaml:11]:
```suggestion
  {{- with .Values.config }}
    {{- toYaml . | nindent 4 }}
  {{- end }}
```
Just nitpicking here...

**Review by bjw-s — APPROVED** [review]
👍

**Pull request merged**